libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx

```diff
@@ -134,8 +134,10 @@ export class PluggableComboChart extends PluggableBaseChart {
             uiConfig: this.getUiConfig(),
         };
 
+        const isMoreThanOneMeasure = findBucket(referencePoint.buckets, BucketNames.MEASURES)?.items.length > 1;
         this.defaultControlsProperties = {
-            stackMeasures: this.isStackMeasuresByDefault(),
+            stackMeasures:
+                this.isStackMeasuresByDefault() && isMoreThanOneMeasure,
```

Comment:
Can you please explain why this isMoreThanOneMeasure not part of the function itself? I see it's called from other places so it would have some other effect, but now it seems to me that it would be out of sync with getUiConfig() for example. But maybe that's intended.